recipe/recipe.yaml

```diff
@@ -37,71 +34,93 @@ source:
+  number: ${{ build_number }}
 
-requirements:
-  build:
-    - if: build_platform != target_platform
-      then:
+outputs:
+  - package:
+      name: polars
+    build:
+      script: echo "No build step is needed"
+      string: ${{ polars_variant | replace('-', '_') }}_h${{ hash }}_${{ build_number }}
+      variant:
+        down_prioritize_variant: ${{ 0 if polars_variant == 'default' else 1 }}
+    requirements:
+      run:
+        - ${{ pin_subpackage('polars-' ~ polars_variant, exact=True) }}
+    tests:
+      - python:
+          pip_check: true
+          imports:
+            - polars
+          python_version: ${{ python_min }}.*
+      - script:
+          - python -c "from polars import DataFrame"
+  - package:
+      name: polars-${{ polars_variant }}
+    build:
+      skip: not (match(python, python_min ~ ".*") and is_abi3)
+      python:
+        version_independent: true
+    requirements:
+      build:
+        - if: build_platform != target_platform
+          then:
+            - python
+            - cross-python_${{ target_platform }}
+            - crossenv
+            - maturin >=1.3.2,<2
+        - ${{ compiler('c') }}
+        - ${{ compiler('cxx') }}
+        - ${{ stdlib('c') }}
+        - ${{ compiler('rust') }}
+        - if: build_platform == 'win-64'
+          then:
+            - posix
+        - cmake
+        - if: unix
+          then:
+            - make
+        - cargo-bundle-licenses
+        # TODO: remove once https://github.com/prefix-dev/rattler-build/issues/1000 is resolved
+        - if: target_platform == 'linux-64' or target_platform == 'osx-64'
+          then:
+            - sed
+        - if: target_platform == 'win-64'
+          then:
+            - m2-sed
+      host:
         - python
-        - cross-python_${{ target_platform }}
-        - crossenv
+        - python-abi3
+        - pip
         - maturin >=1.3.2,<2
-    - ${{ compiler('c') }}
-    - ${{ compiler('cxx') }}
-    - ${{ stdlib('c') }}
-    - ${{ compiler('rust') }}
-    - if: build_platform == 'win-64'
-      then:
-        - posix
-    - cmake
-    - if: unix
-      then:
-        - make
-    - cargo-bundle-licenses
-    # TODO: remove once https://github.com/prefix-dev/rattler-build/issues/1000 is resolved
-    - if: target_platform == 'linux-64' or target_platform == 'osx-64'
-      then:
-        - sed
-    - if: target_platform == 'win-64'
-      then:
-        - m2-sed
-  host:
-    - python
-    - python-abi3
-    - pip
-    - maturin >=1.3.2,<2
-  run:
-    - python
-    - numpy >=1.16.0
-    - if: python <3.11
-      then:
-        - typing_extensions >=4.0.0
-    - if: python >=3.10
-      then:
-        - packaging
-  ignore_run_exports:
-    from_package:
-      - cross-python_${{ target_platform }}
+      run:
+        - python
+        - numpy >=1.16.0
+        - if: python <3.11
+          then:
+            - typing_extensions >=4.0.0
+        - if: python >=3.10
+          then:
+            - packaging
```

Comment:
```suggestion
```

didn't find them anymore in https://github.com/pola-rs/polars/blob/main/py-polars/pyproject.toml so i think we can remove those